core/src/test/java/org/elasticsearch/node/internal/InternalSettingsPreparerTests.java

```diff
@@ -42,6 +44,8 @@
 import static org.hamcrest.Matchers.*;
 
 public class InternalSettingsPreparerTests extends ESTestCase {
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
```

Comment:
ok, then i am fine with it! Maybe we should followup and consider moving it to a base class and we do this stuff in a consistent way everywhere. I bet we will find test bugs.